generated/nidaqmx/_task_modules/triggering/start_trigger.py

```diff
@@ -832,6 +832,23 @@ def retriggerable(self, val):
     def retriggerable(self):
         self._interpreter.reset_trig_attribute(self._handle, 0x190f)
 
+    @property
+    def start_trig_trig_when(self):
```

Comment:
Sorry I didn't notice this before, but the codegen strips `start_trig_` from the other `StartTrigger` properties.

For example:
- `START_TRIG_TYPE` -> `trig_type` (this one might be special, since it still has trig)
- `ANLG_EDGE_START_TRIG_HYST` -> `anlg_edge_hyst`
- `DIG_EDGE_START_TRIG_SRC` -> `dig_edge_src`

So I think these new properties should be `time_when`.

This will likely require another round of scrapigen changes, so it's ok to do this in a separate PR (maybe with the rest of the time trigger attributes).

Reply:
I will address this in a separate PR. Thanks.

Reply:
Addressed in https://github.com/ni/grpc-device-scrapigen/pull/216